src/main.c

```diff
@@ -3181,6 +3181,12 @@ static void *run_loop(void *_thread_index)
             listeners[i].http3.ctx.accept_ctx = &listeners[i].accept_ctx;
             listeners[i].http3.ctx.send_retry = listener_config->quic.send_retry;
             listeners[i].http3.ctx.qpack = listener_config->quic.qpack;
+            if (conf.globalconf.http3.handshake_timeout_rtt_multiplier > 0)
+                listener_config->quic.ctx->handshake_timeout_rtt_multiplier =
+                    conf.globalconf.http3.handshake_timeout_rtt_multiplier;
+            if (conf.globalconf.http3.max_initial_handshake_packets > 0) {
+                listener_config->quic.ctx->max_initial_handshake_packets = conf.globalconf.http3.max_initial_handshake_packets;
+            }
```

Comment:
`listener_config` points to a variable shared across all the threads and therefore should better be adjusted outside of `run_loop` which is a function that is being run on every worker thread. FWIW, we have a block that says "apply HTTP/3 global configuraton to the listeners" in `main`, probably that's the place that we want to make these adjustments.

Note: this comment applies only if we are to retain these config knobs as global (rather than making them part of the `quic` mapping).

Reply:
Ah makes sense. I think this is addressed by moving these configs under `quic` node. (479c20592)